tests/test_maya_memvid_full.py

```
@@ -0,0 +1,245 @@
+        print("🧹 Cleaning up resources...")
+        
+        # Cleanup LLM client
+        if llm is not None:
+            try:
+                if hasattr(llm, 'close'):
+                    llm.close()
+                elif hasattr(llm, 'shutdown'):
+                    llm.shutdown()
+                elif hasattr(llm, '__del__'):
+                    del llm
+            except Exception as cleanup_error:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)